renderMarkdown.js

```diff
@@ -0,0 +1,169 @@
+const marked = require("marked");

```

Comment:
It'd be better t pass in the value of LaTeXinMD into the renderMarkdown function. That way the calling context can decide where LaTeX handling is required and the renderMarkdown function becomes cleaner.